src/components/Dissolution/CustodianOfRecords.vue

```diff
@@ -485,26 +486,19 @@ export default class CustodianOfRecords extends Mixins(CommonMixin) {
 }
 
 // Vuetify styling overrides
-::v-deep .v-text-field .v-label {
+:deep(.v-text-field .v-label) {
   font-weight: normal;
   font-size: $px-16;
   color: $gray7;
   letter-spacing: -0.2px;
 }
 
-::v-deep .v-input--selection-controls .v-input__slot > .v-label {
+:deep(.v-input--selection-controls .v-input__slot > .v-label) {
   font-size: $px-16;
   color: $gray7;
   font-weight: normal;
 }
 
-// italicize delivery instructions and remove top margin
-::v-deep .base-address .address-block .delivery-instructions {
-  font-size: $px-14;
-  font-style: italic;
-  margin-top: 0.5rem !important;
-}
```

Comment:
This is no longer needed here or in other UI components that use BaseAddress because these styles are now in BaseAddress itself.